### DISCUSSION TITLE
RetryCount is not working

### DISCUSSION THREAD

**Discussion by JulAlx** [radsecproxy/radsecproxy#151] [Q&A] [answered]
The application does not accept an integer for 'RetryCount'.

`error in block Server FreeRadius, value of option RetryCount is 5, must be 0-0
configuration error`

**Comment by fmauchle** [comment, marked as answer]
I presume you are using `type tls` or `type tcp`. In this case, this is by design and `RetryCount` must always be 0. Since TCP and TLS are reliable transports, there is no sense in resending a request. The transport will either have delivered the request or the connection would fail, while the server would simply ignore any duplicate requests.

RFC [6613 Section 2.6.1](https://www.rfc-editor.org/rfc/rfc6613#section-2.6.1) explicitly states:
> implementations MUST NOT retransmit RADIUS request packets
